crowbar_framework/app/views/nodes/_buttons_power.html.haml

```diff
@@ -1,10 +1,25 @@
-- if @node.admin?
-  - if @node.bmc_set?
-    = link_to "Identify", hit_node_path(@node.handle, "identify"), :class => "btn btn-default", "data-remote" => true
-- else
-  = link_to "Reboot", hit_node_path(@node.handle, "reboot"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
-  = link_to "Shutdown", hit_node_path(@node.handle, "shutdown"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
+.dropdown
+  %button.btn.btn-default.dropdown-toggle{ type: "button", data: { toggle: "dropdown" } }
+    = t(".actions")
+    %span.caret
 
-  - if @node.bmc_set?
-    = link_to "Power on", hit_node_path(@node.handle, "poweron"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
-    = link_to "Identify", hit_node_path(@node.handle, "identify"), :class => "btn btn-default", "data-remote" => true
+  %ul.dropdown-menu{ role: "menu" }
+    - if @node.admin?
+      - if @node.bmc_set?
+        %li{ role: "presentation" }
+          = link_to t(".identify"), hit_node_path(@node.handle, "identify"), role: "menuitem", remote: true
+    - else
+      %li{ role: "presentation" }
+        = link_to t(".reboot"), hit_node_path(@node.handle, "reboot"), role: "menuitem", remote: true
+      %li{ role: "presentation" }
+        = link_to t(".hardreboot"), hit_node_path(@node.handle, "hardreboot"), role: "menuitem", remote: true
+      %li{ role: "presentation" }
+        = link_to t(".shutdown"), hit_node_path(@node.handle, "shutdown"), role: "menuitem", remote: true
+      %li{ role: "presentation" }
+        = link_to t(".hardshutdown"), hit_node_path(@node.handle, "hardshutdown"), role: "menuitem", remote: true
```

Comment:
Actually: the identify bits should probably not be in the dropdown, it should stay a separate button.